### PR TITLE
Clean up uiflag.txt

### PR DESCRIPTION
**Pull request by jtrumbull**

### CODE REVIEW

File: d2bs/kolbot/sdk/uiflag.txt [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+0x07    New skill button enabled
+0x08	Npc menu open
+0x09	Esc menu open
+0x0A	Automap is open
+0x0B	Configure controls screen open
+0x0C	Npc trade window open
+0x0D	Holding alt to show items on gound
+0x0E	Npc choose item menu open (imbue, socket, personalize, etc)
+0x0F	Quest log open
+0x10    ? Assume help button visible
```
Comment (theBGuy) [inline d2bs/kolbot/sdk/uiflag.txt:16]: 0x10 is Key to the Cairn Stones Screen

File: d2bs/kolbot/sdk/uiflag.txt [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
-0x0A	Automap is on
-0x0B	config controls
-0x0C	Shop open at NPC
-0x0D	alt show items
-0x0E	cash
-0x0F	quest
-0x10
-0x11	questlog button
+0x01    Inventory screen open
+0x02	Character screen open
```
Comment (theBGuy) [inline d2bs/kolbot/sdk/uiflag.txt:2]: 0x02 the original statement Character Stat screen is clearer than Character Screen

**Review by theBGuy — CHANGES_REQUESTED** [review]
See comments, everything else looks good

**Review by theBGuy — APPROVED** [review]

**Comment by jtrumbull** [comment]
Wonderful

**Pull request merged**